libs/langchain/langchain/llms/ollama.py

```diff
@@ -89,11 +89,15 @@ class _OllamaCommon(BaseLanguageModel):
     to more diverse text, while a lower value (e.g., 0.5) will
     generate more focused and conservative text. (Default: 0.9)"""
 
+    format: Optional[str] = None
+    """Specify the format of the output (e.g., JSON)"""
```

Comment:
```suggestion
    """Specify the format of the output (e.g., json)"""
```

`json` may be a better hint/example since that's the only value the api will accept